### DISCUSSION TITLE
Not showing all the apps

### DISCUSSION THREAD

**Discussion by arkodeepg** [Flow-Launcher/Flow.Launcher#1165] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/43224169/166140989-04eca266-e84b-4a73-ada2-32bc9202f7de.png)

I changed the file suffixes while trying to add a program which I was not able to find through search (afterward i found out the add button) and now it is not showing all the apps that are available

**Comment by taooceros** [comment, marked as answer]
Sorry for the late reply. You can change it back to `appref-ms;exe;lnk`
